doc_source/update-cluster.md

```diff
@@ -180,24 +180,244 @@ The cluster update should finish in a few minutes\.
 
 ------
 
-1. Patch the `kube-proxy` daemonset to use the image that corresponds to your cluster's Region and current Kubernetes version \(in this example, `1.16.8`\)\.    
-[\[See the AWS documentation website for more details\]](http://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html)
-
-   First, retrieve your current `kube-proxy` image:
-
+Update `kube-proxy` on Worker nodes:
+- Check existing `kube-proxy` DaemonSet configuration on the cluster,
    ```
-   kubectl get daemonset kube-proxy --namespace kube-system -o=jsonpath='{$.spec.template.spec.containers[:1].image}'
+   kubectl get ds kube-proxy -n kube-system -o jsonpath={.spec.template.spec.containers[0].command}
```

Comment:
use full verbs instead of shorthands sed -i s/ds/daemonset/g and `-n` to `--namespace`